src/app/app.component.ts

```diff
@@ -169,9 +170,16 @@ export class AppComponent implements OnInit, OnDestroy {
             );
           }
 
+          let themeSwitcherUpdates: NextObserver<SkyThemeSettings> = {
+            next: () => {}
+          };
+          this.themeSvc!.settingsChange.subscribe((value) => {
```

Comment:
Looking at the supporting code in the Host service, I'm thinking this should probably go after the call to `setupThemeSwitcher()` in the case that `settingsChange` already has a value and could push an update before `themeSwitcherUpdates.next()` is assigned by Host.